pkg/kev/converter/kubernetes/project_service.go

```diff
@@ -146,7 +126,7 @@ func (p *ProjectService) workloadType() string {
 			workloadType)
 	}
 
-	return workloadType
```

Comment:
I guess no need to change the return value here?

Reply:
I reverted this. to use workloadType, this was left because of copy paste. I brought back the logging but forgot to change this last return